browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1447,6 +1456,18 @@ until they verify, or until 90 days have passed.
       <message name="IDS_PRICE" desc="Binance convert item text">
         Price
       </message>
+      <message name="IDS_PRICE_USD" desc="Text amount for USD">
```

Comment:
These are the temp values as it's only used to test UI. it would be removed once we have right information.